.github/workflows/portage-stable-packages-list

```diff
@@ -363,8 +364,17 @@ profiles
 #
 # scripts
 
+sec-policy/selinux-base
+sec-policy/selinux-base-policy
+sec-policy/selinux-container
+sec-policy/selinux-dbus
+sec-policy/selinux-sssd
+sec-policy/selinux-unconfined
+
+
 sys-apps/acl
 sys-apps/attr
+sys-apps/checkpolicy
```

Comment:
One last thing - could you add `sys-apps/semodule-utils` back to the list too? I remember that it used to be in the automation list, but I dropped it as the update broke with the older version of selinux packages. Now it should be fine to bring it back.

This change should not require the CI rerun as it does not affect the generated images at all.